Source/CMakeLists.txt

```diff
@@ -15,6 +15,9 @@ set_property(CACHE AUDIO_MODULE PROPERTY STRINGS OpenAudio FMOD Null)
 set(PHYSICS_MODULE "PhysX" CACHE STRING "Physics backend to use.")
 set_property(CACHE PHYSICS_MODULE PROPERTY STRINGS "PhysX" "Null")
 
+# include imgui plugin will be default false.
+set(INCLUDE_IMGUI_PLUGIN OFF CACHE BOOL "Include BsfImgui submodule plugin")
```

Comment:
Could we rename the property to something like `BSF_PLUGIN_IMGUI`. I'm hoping to add `BSF` prefix to all the cmake properties eventually, so perhaps best to start now. And and we move the `PLUGIN` naming forward, so when sorted by name, we have all the plugin related stuff grouped together, when printed by cmake (or in CMake GUI). Since there are no categories it will help distinguish various cmake properties as their number grows.

Reply:
done.